noah/src/main/resources/crawler/README.txt

```diff
@@ -0,0 +1,51 @@
+Run the setupScrapy.sh only once to setup the scrapy-splash environment
```

Comment:
Can you make it to a README.md file? `marddown` format will looks better even on github.